aio_pika/patterns/rpc.py

```diff
@@ -404,13 +405,33 @@ class JsonRPC(RPC):
     CONTENT_TYPE = "application/json"
 
     def serialize(self, data: Any) -> bytes:
-        return self.SERIALIZER.dumps(data, ensure_ascii=False, default=repr)
+        return self.SERIALIZER.dumps(
+            data, ensure_ascii=False, default=repr).encode('ascii')
+
+    def deserialize(self, data: Any) -> bytes:
+        res = super().deserialize(data)
+        if isinstance(res, dict) and "error" in res:
+            cls = locate(res['error']['type'])
+            if not cls:
+                def exception_constructor(self, message, args):
+                    self.message = message
+                    self.args = args
+
+                cls = type(res['error']['type'], (Exception, ), {
+                    '__init__': exception_constructor
+                })
+
+            res = cls(res['error']['message'], res['error']['args'])
+        return res
 
     def serialize_exception(self, exception: Exception) -> bytes:
         return self.serialize(
             {
                 "error": {
-                    "type": exception.__class__.__name__,
+                    "type": f'{exception.__module__}.'
```

Comment:
fixed based on your recommendation

Reply:
I left the __module__ in the message in case the exception is handled by Python client and the client wants to know the exact origin of the exception and maybe recreated it in higher layers

Reply:
`inspect.getmodule` will looks better I hope.

Reply:
Why `inspect.getmodule(exception).__name__` is better than `exception.__module__` ?